opennms-webapp-rest/src/test/java/org/opennms/web/rest/v1/RemotePollerAvailabilityRestServiceIT.java

```diff
@@ -87,6 +88,10 @@
 @JUnitConfigurationEnvironment
 @JUnitTemporaryDatabase
 @Transactional
+/**
+ * Ignore for now, since the remote poller with minion stuff is still in development
+ */
+@Ignore
```

Comment:
Please add an `TODO cpape` so we can find this later on.

Reply:
done